core/src/main/java/org/renjin/primitives/packaging/Namespace.java

```diff
@@ -95,27 +93,27 @@ public String getCompatibleName() {
     return nativeSymbolMap;
   }
 
-  public SEXP getEntry(Symbol entry) {
-    SEXP value = namespaceEnvironment.getVariable(entry);
+  public SEXP getEntry(Context context, Symbol entry) {
```

Comment:
Active bindings in namespace environments seem like a really bad idea. Is this even possible in GNU R, and if so, can we safely prevent this behavior so namespace lookup continues to be free of side effects?